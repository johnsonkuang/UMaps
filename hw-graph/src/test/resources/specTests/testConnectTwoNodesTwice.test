# Test two different connections between nodes
# Test whether two edges with different labels can be made between
# two nodes in a graph

# Create graph
CreateGraph graph1

# Add two nodes
AddNode graph1 A
AddNode graph1 B

# Add two edges with A as the parent and B as the child
AddEdge graph1 A B e1
AddEdge graph1 A B e2

# List the children of A, should contain B twice
ListChildren graph1 A