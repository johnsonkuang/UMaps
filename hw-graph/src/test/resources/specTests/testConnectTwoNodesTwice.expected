# Test two different connections between nodes
# Test whether two edges with different labels can be made between
# two nodes in a graph

# Create graph
created graph graph1

# Add two nodes
added node A to graph1
added node B to graph1

# Add two edges with A as the parent and B as the child
added edge e1 from A to B in graph1
added edge e2 from A to B in graph1

# List the children of A, should contain B twice
the children of A in graph1 are: B(e1) B(e2)